### ISSUE TITLE
[Roster View > Add a family member through creating a new child account]: talkback focus does not land on the first control of the "Create account for child" page after activating the 'Create child account' button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14009]
Steps to reproduce:
Launch NVDA
1. Turn on Talkback, Launch Family Safety application and login with above credential.
2. Navigate to 'Add a family member' button using swipe gesture and activate it using double tap.
3. Navigate to 'Create a child account' button using swipe gesture and activate it using double tap.
4. Observe the talkback focus and announcement.

Actual behavior:
Talkback focus does not land on the first control of the "Create account for child" page after activating the 'Create child account' button. also, talkback does not announce "Create account for child" page information.

Expected behavior:
Talkback focus should land on the first controls of the Create account for child" page and talkback should announce "Create account for child" page information.

https://user-images.githubusercontent.com/93735775/184422326-5aee0b9b-3d2c-47d8-b259-1b6b69392fc3.mp4

NVDA logs, crash dumps and other attachments:

**Comment by cary-rowen** [comment]
You seem to be talking about talkback all the time, and here is NVDA Issue, which is suggested to be closed.

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- This appears to be a talkback issue
- Please fill out the issue template fully.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by msftedad** [comment]
Hello Team,

This is an invalid issue, hence closing it.
